packages/ariakit-test/readme.md

```
@@ -17,3 +17,221 @@ npm i @ariakit/test
 ## Contributing
 
 Follow the instructions on the [contributing guide](https://github.com/ariakit/ariakit/blob/main/contributing.md).
+
+<!-- ariakit-docs:start -->
+
+## API reference
+
+#### `blur`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)